src/include/migraphx/env.hpp

```diff
@@ -33,18 +33,20 @@ inline namespace MIGRAPHX_INLINE_NS {
 
 // Declare a cached environment variable
 #define MIGRAPHX_DECLARE_ENV_VAR(x)               \
-    struct x                                      \
+    struct env_variable_##x                       \
     {                                             \
         static const char* value() { return #x; } \
-    }; // NOLINT
+    };                                            \
+                                                  \
+    inline constexpr auto x = env_variable_##x{};
```

Comment:
Why is this needed for windows?

Reply:
However, I will turn it back if you prefer the old way.

Reply:
Actually, considering the same optimization options, both versions resolve to the same machine code. The compiler is doing the right job anyway: https://godbolt.org/z/Yr7P1P3KP

Let me know if you want to revert the change or keep it. I will remove it from this PR since it is not required.